raidprotect/src/interaction/command/moderation/kick.rs

```diff
@@ -66,9 +66,10 @@ impl KickCommand {
         let author_id = interaction.author_id().context("missing author_id")?;
 
         let user = self.user.resolved;
+        let lang = Lang::from(&interaction.clone().locale.unwrap() as &str);
```

Comment:
We should not unwrap the value, use `context` instead (from the `anyhow::Context` trait).

```suggestion
        let lang = Lang::from(&interaction.locale.context("missing locale")?);
```